atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -269,35 +289,45 @@ private static SerializableTransactionManager create(
         return transactionManager;
     }
 
-    private static SweepBatchConfig getSweepBatchConfig(AtlasDbConfig config) {
+    private static void validateRuntimeConfig(
+            java.util.function.Supplier<java.util.Optional<AtlasDbRuntimeConfig>> runtimeConfig) {
+        if (!runtimeConfig.get().isPresent()) {
+            log.warn("AtlasDB now supports live reloadable configs. Please use the 'atlas-runtime' block for specifying"
+                    + " live-reloadable atlas configs. Default to using configs specified on the 'atlas' block");
```

Comment:
drive-by: note that atlasdb doesn't know which config key each config appears in, so if anyone deviates this error message will be very confusing. The atlas codebase used to have a bunch of error messages which assumed atlas was inside gotham - which obviously after a while became false.